distro/src/bin/atlas_start.py

```diff
@@ -143,13 +135,10 @@ def main():
     web_app_path = os.path.join(web_app_dir, "atlas")
     if (mc.isCygwin()):
         web_app_path = mc.convertCygwinPath(web_app_path)
-    if not is_setup:
-        start_atlas_server(atlas_classpath, atlas_pid_file, jvm_logdir, jvm_opts_list, web_app_path)
-        mc.wait_for_startup(confdir, 300)
-        print "Apache Atlas Server started!!!\n"
-    else:
-        process = mc.java("org.apache.atlas.web.setup.AtlasSetup", [], atlas_classpath, jvm_opts_list, jvm_logdir)
```

Comment:
To go along with this, the current script fails. And this line needs to be changed to:
```py
process = mc.java("org.apache.atlas.web.setup.AtlasSetupModule", [], atlas_classpath, jvm_opts_list, jvm_logdir)
```
otherwise, `-setup` fails.

Reply:
I don't believe there is a AtlasSetupModule anymore: https://github.com/apache/atlas/tree/master/webapp/src/main/java/org/apache/atlas/web/setup

The purpose of this PR is to entirely remove the -setup option as it doesn't seem to be needed anymore.